app/models/compute_resources/foreman/model/gce.rb

```diff
@@ -100,6 +116,14 @@ def self.provider_friendly_name
       "Google"
     end
 
+    def interfaces_attrs_name
+      "network_interfaces"
```

Comment:
very tiny nitpick - use a symbol. this one will return a new string instance every time you call this method.